### PR TITLE
New package: QMPlay2-18.07.03

### PR DESCRIPTION
**Pull request by abenson**
Closes https://github.com/void-linux/void-packages/issues/514

### CODE REVIEW

**Comment by jnbr** [comment]
ffmpeg4 (https://github.com/void-linux/void-packages/pull/474) just got merged. I would delay this until there is a QMPlay2 which supports ffmpeg4.

**Comment by notramo** [comment]
@abenson there is a new release.

**Comment by abenson** [comment]
Nice, with the FFmpeg 4.x patches.

File: srcpkgs/QMPlay2/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+version=18.07.03
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="alsa-lib-devel ffmpeg-devel libass-devel libcdio-devel libgme-devel
+ libsidplayfp-devel libva-glx-devel libXv-devel pulseaudio-devel
+ qt5-svg-devel qt5-tools-devel qt5-x11extras-devel taglib-devel"
+short_desc="Video and audio player which can play most formats and codecs"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-3.0-or-later"
```
Comment (cr6git) [inline srcpkgs/QMPlay2/template]: I think it's `LGPL-3.0-only`

File: srcpkgs/QMPlay2/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+homepage="http://zaps166.sourceforge.net/?app=QMPlay2"
+distfiles="https://github.com/zaps166/QMPlay2/archive/${version}.tar.gz"
+checksum=e9dd6caa74e0824d950c0d11b2ed71e799f0a02c375b49ff635498133ab6b45e
```
Comment (cr6git) [inline srcpkgs/QMPlay2/template]: According to https://github.com/zaps166/QMPlay2/releases , the source tarball is:
````
distfiles="https://github.com/zaps166/QMPlay2/releases/download/${version}/QMPlay2-src-${version}.tar.xz"
checksum=78cdadea21728f7a902ae240aae15481bcad2b9f34598c20ee7711feeff79122
````

File: srcpkgs/QMPlay2/template (hunk 1 above)
Comment (abenson) [reply srcpkgs/QMPlay2/template]: Good catch

File: srcpkgs/QMPlay2/template (hunk 2 above)
Comment (abenson) [reply srcpkgs/QMPlay2/template]: I chose the other one to avoid having to change wrksrc, but that one works fine

**Comment by Johnnynator** [comment]
still needs a patch for musl
```
[ 19%] Building CXX object src/qmplay2/CMakeFiles/libqmplay2.dir/libqmplay2_autogen/mocs_compilation.cpp.o
/builddir/QMPlay2-src-18.07.03/src/qmplay2/IPC_Unix.cpp: In member function 'virtual bool IPCSocket::open(QIODevice::OpenMode)':
/builddir/QMPlay2-src-18.07.03/src/qmplay2/IPC_Unix.cpp:101:9: error: 'u_long' does not name a type; did you mean 'ulong'?
   const u_long on = 1;
         ^~~~~~
         ulong
/builddir/QMPlay2-src-18.07.03/src/qmplay2/IPC_Unix.cpp:102:31: error: 'on' was not declared in this scope
   ioctl(m_priv->fd, FIONBIO, &on);
                               ^~
```

**Comment by abenson** [comment]
That was an easy fix. I PR'd it upstream, too

**Comment by notramo** [comment]
@cr6git Could it be merged now?

**Pull request merged**